### PR TITLE
cli (ticdc): refine cdc cli output

### LINKED ISSUE #6651: ticdc exit graceful
### What did you do?

1、 create changefeed
2、enter N

### What did you expect to see?

The changefeed create command should not reports a bug and should inform users that this changefeed creating procedure is aborted.

### What did you see instead?

image

### Versions of the cluster

ctl and cluster version is v5.4.2

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #6651 

### What is changed and how it works?
Use `Run` instead of  `RunE` in `cmd` to run the main logic of all clis, and use `util.CheckErr` to handle the return error.
Which can avoid cli prints verbose usage messages even though the error is not a parse error.

#### Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)
before:
image

after:
image

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

File: pkg/cmd/cli/cli_changefeed_query_test.go [hunk 1]
```diff
@@ -57,29 +62,36 @@ func TestChangefeedQueryCli(t *testing.T) {
...
 	out, err := ioutil.ReadAll(b)
 	require.Nil(t, err)
 	// make sure config is printed
 	require.Contains(t, string(out), "config")
 
 	// query failed
 	cfV1.EXPECT().Get(gomock.Any(), "bcd").Return(nil, errors.New("test"))
+	o.simplified = false
+	o.changefeedID = "bcd"
 	os.Args = []string{"query", "--simple=false", "--changefeed-id=bcd"}
```
[the remark this answers is not in the document]
Comment (asddongmen) [reply pkg/cmd/cli/cli_changefeed_query_test.go:95]: good catch~

File: pkg/errors/helper.go [hunk 2]
```diff
@@ -153,3 +153,18 @@ func ToPBError(err error) *pb.Error {
 	pbErr.Message = err.Error()
 	return pbErr
 }
+
+var cliNotPrintError = []*errors.Error{ErrChangefeedCliAborted}
+
+// IsCliNotPrintError returns true if the error should not be printed in cli.
+func IsCliNotPrintError(err error) bool {
```
[the remark this answers is not in the document]
Comment (asddongmen) [reply pkg/errors/helper.go:160]: I think it is better to keep it outside this func as other type of `error` define in this file.

File: pkg/cmd/cli/cli_changefeed_helper.go [hunk 3]
```diff
@@ -44,7 +44,8 @@ func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint
 			return err
 		}
 		if strings.ToLower(strings.TrimSpace(yOrN)) != "y" {
-			return errors.NewNoStackError("abort changefeed create or resume")
+			cmd.Printf("abort changefeed create\n")
+			return cerror.ErrCliAborted.FastGenByArgs("cli changefeed create or resume")
```
Comment (hi-rustin) [inline pkg/cmd/cli/cli_changefeed_helper.go:48]: This message confused me.

Comment (hi-rustin) [reply pkg/cmd/cli/cli_changefeed_helper.go:48]: The above message said this is a `create`, but this message said this is a `create or resume`. Can we indicate it directly?

Comment (asddongmen) [reply pkg/cmd/cli/cli_changefeed_helper.go:48]: both create and resume changefeed cli would call this func

File: pkg/cmd/cli/cli_changefeed_helper.go [hunk 4]
```diff
@@ -30,7 +30,7 @@ const (
 )
 
 // confirmLargeDataGap checks if a large data gap is used.
-func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint64) error {
+func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint64, cli string) error {
```
Comment (hi-rustin) [inline pkg/cmd/cli/cli_changefeed_helper.go:33]:
```suggestion
func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint64, command string) error {
```

**Review by hi-rustin — APPROVED** [review]
nit

**Pull request merged**